src/Hedgehog/Plutus/TestSingleScript.hs

```diff
@@ -47,11 +47,11 @@ import Cardano.Slotting.Time qualified as Cardano
 import PlutusLedgerApi.Common qualified as Plutus
 import PlutusLedgerApi.V2 qualified as Plutus hiding (evaluateScriptCounting)
 
+import Cardano.Simple.Cardano.Class qualified as Fork
```

Comment:
Change these to `Simple` throughout?

Reply:
Oh right, the name Fork doesn't make much sense now. :+1: